### PR TITLE
Fixed providers go modules, examples and consistency.

### PR DESCRIPTION
**Pull request by bwplotka**
Depend on https://github.com/grpc-ecosystem/go-grpc-middleware/pull/291 
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: providers/kit/logger.go [hunk 1]
```diff
@@ -9,14 +9,20 @@ import (
 	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/logging"
 )
 
+// Compatibility check.
+var _ logging.Logger = &Logger{}
+
+// Logger is an go-kit/log logging adapter compatible with logging middlewares.
```
Comment (johanbrandhorst) [inline providers/kit/logger.go:15]: Nit:

```suggestion
// Logger is a go-kit/log logging adapter compatible with logging middlewares.
```

File: providers/logrus/doc.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/*
+`logrus` provides small adapter required to use logrus in logging gRPC middlewares.
```
Comment (johanbrandhorst) [inline providers/logrus/doc.go:2]:
```suggestion
`logrus` provides a small adapter required to use logrus in logging gRPC middlewares.
```

**Comment by bwplotka** [comment]
all addressed @johanbrandhorst  (:

**Comment by bwplotka** [comment]
Let's address potential comments on next PR. :hugs:

**Pull request merged**